src/DocFxTocGenerator/README.md

```diff
@@ -7,15 +7,16 @@ This tool allow to generate a yaml compatible `toc.yml` file for DocFX.
 ```text
 TocGenerator -d <docs folder> [-o <output folder>] [-vsi]
 
--d, --docfolder       Required. Folder containing the documents.
--o, --outputfolder    Folder to write the resulting toc.yml in.
--v, --verbose         Show verbose messages.
--s, --sequence        Use the .order files for TOC sequence. Format are raws of: filename-without-extension
--r, --override        Use the .override files for TOC file name override. Format are raws of: filename-without-extension;Title you want
--i, --index           Auto-generate a file index in each folder.
--g, --ignore          Use the .ignore files for TOC directory ignore. Format are raws of directory names: directory-to-ignore
---help                Display this help screen.
---version             Display version information.
+-d, --docfolder			Required. Folder containing the documents.
```

Comment:
Can you please properly align the text like it was done with the previous text?
Use spaces and not tabs (not sure what's use).
![image](https://github.com/Ellerbach/docfx-companion-tools/assets/8145578/24589789-21a3-43ee-baef-cedb1a56bd7b)

Reply:
Looks like this got merged already, but I'll fix the readme, I had to push the descriptions over to make room for the <depth> parameter...not sure if that's the proper way of documenting param inputs or not